bucket/tinytask.json

```
@@ -0,0 +1,23 @@
+{
+    "homepage": "https://www.tinytask.net/",
+    "description": "A minimalist Windows automation app you can use to record and repeat actions.",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://www.tinytask.net/faq.php"
+    },
+    "url": "https://www.tinytask.net/TinyTask.exe",
+    "version": "1.72",
+    "hash": "9957f2b0e80a03421ff11ab680cb07d111409a03c41540db5a6998ff685f6265",
+    "shortcuts": [
+        [
+            "TinyTask.exe",
+            "TinyTask"
+        ]
+    ],
+    "pre_install": "if (!(Test-Path \"$persist_dir\\TinyTask.ini\")) { New-Item \"$dir\\TinyTask.ini\" -ItemType File | Out-Null }",
+    "persist": "TinyTask.ini",
+    "checkver": "version ([\\w.]+)",
```

`\w` is pretty overkil for just numbers

```suggestion
    "checkver": "version ([\\d.]+)",
```